### PR TITLE
migration allow null for ZDT

### PR DESCRIPTION
**Pull request by devincowan**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by devincowan** [comment]
@pkdash we can avoid ~2hrs downtime for 1.60 deploy if we allow null on the new user_is_active field.
This is so that the active worker that's running 1.59 can still create resources etc _after_ migration is run but _before_ swap to the 1.60 worker.

My thought is that we would have another migration to get rid of the NullBooleanFields (which have been deprecated in newer django) as part of 1.61.

Thoughts?

@martinseul if we get this in, it should allow us to go ZDT for 1.60 release

File: hs_core/migrations/0065_auto_20221004_1707.py [hunk 1]
```diff
@@ -22,12 +22,12 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='contributor',
             name='is_active_user',
-            field=models.BooleanField(default=False),
+            field=models.NullBooleanField(default=False, null=True),
```
Comment (pkdash) [inline hs_core/migrations/0065_auto_20221004_1707.py:25]: I think we need `blank=True` here

File: hs_core/migrations/0065_auto_20221004_1707.py [hunk 2]
```diff
@@ -22,12 +22,12 @@ class Migration(migrations.Migration):
...
         ),
         migrations.AddField(
             model_name='creator',
             name='is_active_user',
-            field=models.BooleanField(default=False),
+            field=models.NullBooleanField(default=False, null=True),
```
Comment (pkdash) [inline hs_core/migrations/0065_auto_20221004_1707.py:30]: I think we need `blank=True` here

**Review by pkdash — CHANGES_REQUESTED** [review]

File: hs_core/migrations/0065_auto_20221004_1707.py (hunk 2 above)
Comment (devincowan) [reply hs_core/migrations/0065_auto_20221004_1707.py:30]: https://docs.djangoproject.com/en/1.11/_modules/django/db/models/fields/#NullBooleanField
Thanks @pkdash looks like null and blank are automatically set for a `NullBooleanField` so I'm actually going to remove the null=True kwarg...

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4812](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4812)

**Pull request closed without merging**